simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/HotspotMetricCollectorsConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package io.prometheus.client.spring;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.prometheus.client.Summary;
+import io.prometheus.client.hotspot.ClassLoadingExports;
+import io.prometheus.client.hotspot.GarbageCollectorExports;
+import io.prometheus.client.hotspot.MemoryPoolsExports;
+import io.prometheus.client.hotspot.StandardExports;
+import io.prometheus.client.hotspot.ThreadExports;
+
+/**
+ * A {@link Configuration} class that imports the Prometheus
+ * JVM management metrics Collectors from the <code>simpleclient_hotspot</code> package.
+ *
+ * @author pidster
+ */
+@Configuration
+public class HotspotMetricCollectorsConfiguration {
```

Comment:
Spring does not need a main for tests, e.g. for Boot, main is just used to call a utility that starts the Spring container - which then handles the rest of the class lifecycle. 

The tests in this module now don't have a main class, because a complete Spring & Servlet container can be started on a random-ish port and run inside the test class by Springs JUnit integration. See `io.prometheus.client.spring.boot.AutoConfigurationTests` for an example.